lib/CL/poclcc_binary.c

```diff
@@ -0,0 +1,443 @@
+#include "poclcc_binary.h"
+
+void poclcc_free(poclcc_global *binary_format){
```

Comment:
Better name might be 'pocl_binary_free' and poclcc_global should be simply pocl_binary ?